src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -0,0 +1,291 @@
+            this.storeForAllData.mergedDiscreteCNADataFilteredByGene &&
+            this.storeForAllData
+                .mergedMutationDataIncludingUncalledFilteredByGene
+        );
+    }
+    public render() {
+        //control the data
+        if (this.isNewStoreReady) {
+            this.addGenomicData(this.alterationFrequencyData);
+            this.getQueryGenes(this.alterationFrequencyData);
+        }
+        if (!this.PathwayMapperComponent) {
+            return null;
+        }
+        return (
+            <div className="pathwayMapper">
+                <div
+                    data-test="pathwayMapperTabDiv"
+                    className="cBioMode"
+                    style={{ width: '99%' }}
+                >
+                    <Row>
+                        <React.Fragment>
+                            {/*
+                                  // @ts-ignore */}
+                            <this.PathwayMapperComponent
+                                isCBioPortal={true}
+                                isCollaborative={false}
+                                genes={this.getQueryGenes(
+                                    this.alterationFrequencyData
+                                )}
+                                cBioAlterationData={
+                                    this.alterationFrequencyData
+                                }
+                                changePathwayHandler={this.handlePathwayChange}
+                                addGenomicDataHandler={
+                                    this.addGenomicDataHandler
+                                }
+                                tableComponent={this.renderTable}
+                                validGenes={this.validGenes}
+                                patientView={true}
+                                //message banner patch will be removed
+                                messageBanner={this.renderBanner}
+                            />
+                            <ToastContainer
+                                closeButton={<i className="fa fa-times" />}
+                            />
+                        </React.Fragment>
+                    </Row>
+                </div>
+            </div>
+        );
+    }
+
+    @computed get storeForAllData(): PatientViewPageStore | undefined {
+        let patientStore: PatientViewPageStore | undefined;
+        if (this.urlWrapperForAllGenes) {
+            return patientStore;
+        } else {
+            return undefined;
+        }
+    }
+
+    /**
+     * Here we clone the "query" field of the main store's URL Wrapper and enhance the cloned query
+     * with additional non-query genes. This new query object is used to fake a new URL Wrapper instance
+     * which is required to initialize a separate ResultsViewStore for the non-query genes.
+     */
+    @computed get urlWrapperForAllGenes(): PatientViewUrlWrapper | undefined {
```

Comment:
I don't think we need a separate URL wrapper for patient view. We can remove this.